VTEX - Catalog API.json

```diff
@@ -6223,35 +5808,19 @@
                     "SKU file"
                 ],
                 "summary": "Get SKU files",
-                "description": "Gets general information about all Files in the SKU. \r\n## Response body example\r\n\r\n```json\r\n[\r\n    {\r\n        \"Id\": 549,\r\n        \"ArchiveId\": 155485,\r\n        \"SkuId\": 310118490,\r\n        \"Name\": \"chimera-cat-quimera-5\",\r\n        \"IsMain\": true,\r\n        \"Label\": \"miau\"\r\n    },\r\n    {\r\n        \"Id\": 550,\r\n        \"ArchiveId\": 155486,\r\n        \"SkuId\": 310118490,\r\n        \"Name\": \"Gato-siames\",\r\n        \"IsMain\": false,\r\n        \"Label\": \"Gato siames\"\r\n    },\r\n    {\r\n        \"Id\": 555,\r\n        \"ArchiveId\": 155491,\r\n        \"SkuId\": 310118490,\r\n        \"Name\": \"Cat-Sleeping-Pics\",\r\n        \"IsMain\": false,\r\n        \"Label\": null\r\n    }\r\n]\r\n```",
+                "description": "Gets general information about all files in the SKU.\r\n\r\n## Permissions\r\n\r\nAny user or [application key](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys) must have at least one of the appropriate [License Manager resources](https://help.vtex.com/en/tutorial/license-manager-resources--3q6ztrC8YynQf6rdc6euk3) to be able to successfully run this request. Otherwise they will receive a status code `403` error. These are the applicable resources for this endpoint:\r\n\r\n| **Product** | **Category** | **Resource** |\r\n| --------------- | ----------------- | ----------------- |\r\n| Catalog | Content | **SKUs** |\r\n\r\nThere are no applicable [predefined roles](https://help.vtex.com/en/tutorial/predefined-roles--jGDurZKJHvHJS13LnO7Dy) for this resource list. You must [create a custom role](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc#creating-a-role) and add at least one of the resources above in order to use this endpoint. To learn more about machine authentication at VTEX, see [Authentication overview](https://developers.vtex.com/docs/guides/authentication).\r\n\r\n>\u2757 To prevent integrations from having excessive permissions, consider the [best practices for managing app keys](https://help.vtex.com/en/tutorial/best-practices-application-keys--7b6nD1VMHa49aI5brlOvJm) when assigning License Manager roles to integrations.",
```

Comment:
```suggestion
                "description": "Retrieves general information about all files in the SKU.\r\n\r\n## Permissions\r\n\r\nAny user or [application key](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys) must have at least one of the appropriate [License Manager resources](https://help.vtex.com/en/tutorial/license-manager-resources--3q6ztrC8YynQf6rdc6euk3) to be able to successfully run this request. Otherwise they will receive a status code `403` error. These are the applicable resources for this endpoint:\r\n\r\n| **Product** | **Category** | **Resource** |\r\n| --------------- | ----------------- | ----------------- |\r\n| Catalog | Content | **SKUs** |\r\n\r\nThere are no applicable [predefined roles](https://help.vtex.com/en/tutorial/predefined-roles--jGDurZKJHvHJS13LnO7Dy) for this resource list. You must [create a custom role](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc#creating-a-role) and add at least one of the resources above in order to use this endpoint. To learn more about machine authentication at VTEX, see [Authentication overview](https://developers.vtex.com/docs/guides/authentication).\r\n\r\n>\u2757 To prevent integrations from having excessive permissions, consider the [best practices for managing app keys](https://help.vtex.com/en/tutorial/best-practices-application-keys--7b6nD1VMHa49aI5brlOvJm) when assigning License Manager roles to integrations.",
```